src/main/java/org/openstreetmap/atlas/checks/event/CheckFlagEvent.java

```diff
@@ -217,4 +219,35 @@ public String toString()
     {
         return this.toGeoJsonFeatureCollection().toString();
     }
+
+    public String asLineDelimitedGeoJsonFeatures()
+    {
+        return asLineDelimitedGeoJsonFeatures((jsonObject) ->
+        {
+        });
+    }
+
+    public String asLineDelimitedGeoJsonFeatures(final Consumer<JsonObject> jsonMutator)
+    {
+        final JsonObject flagGeoJsonFeature = flag.asGeoJsonFeature();
```

Comment:
It's strange. If you look at `config/checkstyle/checkstyle.xml`, you can see it's configured to complain when missing `this` on a field.
```
.
.
.
<module name="RequireThis">
             <property name="checkMethods" value="false"/>
</module>
.
.
.
```
But for whatever reason it doesn't actually end up complaining if you leave off `this`.

To clarify the issue I introduced earlier: my save-action was incorrectly adding `this` to method calls as well, which you can see is explicitly disabled in the checkstyle config. I fixed all those extra `this` on the methods since `this` on methods is redundant anyway.

I think there's an argument to be made for using `this` on fields, but I don't really think it's worth aggravating over.

Just be extra careful when adding local variables that you aren't shadowing a member variable :)